packages/core/src/api/clipboard/fromClipboard/handleFileInsertion.ts

```diff
@@ -143,11 +151,22 @@ export async function handleFileInsertion<
 
         const blockInfo = getBlockInfo(posInfo);
 
-        insertedBlockId = editor.insertBlocks(
-          [fileBlock],
-          blockInfo.bnBlock.node.attrs.id,
-          "after"
-        )[0].id;
+        if (
+          blockInfo.isBlockContainer &&
```

Comment:
might be nice to extract this somewhere as I think we use it in more than 1 place